src/kata6.joinNames.js

```diff
@@ -1,5 +1,6 @@
 const joinNames = (namesObj) => {
```

Comment:
👌 A comment on the variable naming, `x` makes sense in a situation where we're talking about spacial coordinates, but on an array of names not as much. Maybe we can use `name` instead?